src/main/java/emt/util/EMTCraftingAspects.java

```diff
@@ -83,6 +83,8 @@ public class EMTCraftingAspects {
             .add(Aspect.MECHANISM, 32);
     public static AspectList wandChargeFocus = new AspectList().add(Aspect.ENERGY, 48).add(Aspect.AURA, 32)
             .add(Aspect.EXCHANGE, 48).add(Aspect.GREED, 64);
+    public static AspectList maintenanceFocus = new AspectList().add(Aspect.CLOTH, 64).add(Aspect.CRAFT, 32)
```

Comment:
Why Pannus? (cloth) That doesn't really make sense here? double the instrumentum and praecantatio instead, possibly add permutatio (32-64) since that revolves around exchange, which is a sort of renewal in a way.